### PR TITLE
basic nav implemented

### PR DESCRIPTION
**Pull request by johnmarksinclair**

### CODE REVIEW

File: navigation/index.tsx [hunk 1]
```diff
@@ -64,32 +64,18 @@ function BottomTabNavigator() {
       }}>
       <BottomTab.Screen
         name="TabOne"
```
Comment (lexesj) [inline navigation/index.tsx:66]:
```suggestion
        name="Profile"
```

Nit: Shouldn't the names (and other references to `TabOne` and `TabTwo`) also be changed to `Profile` and `Settings` to match the new screens?

**Review by lexesj — APPROVED** [review]

**Comment by DanielNugent** [comment]
image

Should the bottom navigation not be present when selecting an image? Would like some input from @lexesjan @dabreadman

**Review by DanielNugent — APPROVED** [review]

**Comment by lexesj** [comment]
> image
> 
> Should the bottom navigation not be present when selecting an image? Would like some input from @lexesjan @dabreadman

I think its fine to not have the navbar when selecting an image. Not too pushed either way though.

**Comment by dabreadman** [comment]
> image
> 
> Should the bottom navigation not be present when selecting an image? Would like some input from @lexesjan @dabreadman

Question: In what user stories would the navbar be present when selecting a picture to upload?

**Comment by DanielNugent** [comment]
Agreed lgtm

**Pull request merged**